.gitlab-ci.yml

```diff
@@ -22,7 +22,7 @@ include:
     file: '.gitlab-ci-github-status-updates.yml'
 
 test:unit:
-  image: golang:1.11
+  image: golang:1.13
```

Comment:
I believe golang:1.13 is the default (see https://github.com/mendersoftware/mendertesting/blob/master/.gitlab-ci-check-golang-unittests.yml#L30), maybe it makes sense to stick to the default?